### PR TITLE
Frame shape Icon can be selected.

### PR DESCRIPTION
**Pull request by YuuyaFujiwara**
Purpose: User can select frame shape icon in Data Flight Mode.
How to Use：
  1) make frame icon file ( *.png ). Nose is above. 　Background color is transparent. 
Appropriate dimensions( around 50x50 to 100x100pixels. Will displayed same size )
  2) Open [CONFIG/TUNING] screen in MissionPlanner. and find [Frame Shape].
  3) Browse Select user made png file. user can select multi files.
  4) selected png will displayed in Data Flight Screen.

Excuse : Only below frames. : Multicopter / Rover / Boat / Heri  /Plane and Caxial(single).

Description of code:
  I edited GCSViews\ConfigurationVIew\ConfigPlanner.cs and related some files.
  And ExtLig\Utilities\Settings.cs. for new settings.

 I added new class [GMapMarker_IconSelectable].  Inherit [GMapMarker] . and base class of each frame classes.( ex. GMapMakerQuad. GmapMakerRover. ... )
Of course, I edited frame class files.( GmapMakerQuad.cs, and more....)
 
In Plane mode ( GMapMakerPlane.cs), user should select six png files. for different images are displayed for each MAV.sysid.

### CODE REVIEW

**Comment by muramura** [comment]
I think that it is better to summarize the commit being worked on.

**Comment by YuuyaFujiwara** [comment]
Summarized the commit by [git rebase].

**Comment by YuuyaFujiwara** [comment]
Rebased to upstream/master #1982

File: ExtLibs/ProjNet/CoordinateSystems/Ellipsoid.cs [hunk 1]
```diff
@@ -142,7 +142,7 @@ public static Ellipsoid Clarke1880
 			get
 			{
 				return new Ellipsoid(20926202, 0, 297, true, LinearUnit.ClarkesFoot, "Clarke 1880", "EPSG", 7034, "Clarke 1880", String.Empty,
-					"Clarke gave a and b and also 1/f=293.465 (to 3 decimal places).  1/f derived from a and b = 293.4663077�");
+					"Clarke gave a and b and also 1/f=293.465 (to 3 decimal places).  1/f derived from a and b = 293.4663077.");
```
Comment (tajisoft) [inline ExtLibs/ProjNet/CoordinateSystems/Ellipsoid.cs]: I think this change has nothing to do with the icon. So I think it should be another PR or remove this change from commit.

Comment (muramura) [reply ExtLibs/ProjNet/CoordinateSystems/Ellipsoid.cs]: I copied this part from github and is coping with paste. The character "..." seems to be garbled.
Clarke gave a and b and also 1 / f = 293.465 (to 3 decimal places). 1 / f derived from a and b = 293.4663077 ... "

Comment (YuuyaFujiwara) [reply ExtLibs/ProjNet/CoordinateSystems/Ellipsoid.cs]: I see...
I'll below:
  1) close this PR.
  2) create new PR for fix frabled char.
  3) reopen this pr ( or create new PR same as  this. ).
